ui/components/app/simulation-preview/asset-pill.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Text } from '../../component-library';
+import {
+  AlignItems,
+  BackgroundColor,
+  BorderRadius,
+  Display,
+  FlexDirection,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import { AssetInfo } from './types';
+
+const EthAssetPill: React.FC = () => (
+  <Text
+    display={Display.Flex}
+    flexDirection={FlexDirection.Row}
+    borderRadius={BorderRadius.pill}
+    alignItems={AlignItems.center}
+    backgroundColor={BackgroundColor.backgroundAlternative}
+    gap={1}
+    style={{
+      padding: '0px 8px 0 4px',
+    }}
+    variant={TextVariant.bodyMd}
+  >
+    <img alt="ETH logo" src="./images/eth_badge.svg" width="18px" />
```

Comment:
Do we not have any existing constant for this URL or any existing components that encapsulate it?

Reply:
I've re-pinged the design team.

Reply:
Okay, done in 2nd PR>